java/arcs/tools/Recipe2Plan.kt

```diff
@@ -0,0 +1,55 @@
+package arcs.tools
+
+import arcs.core.storage.api.DriverAndKeyConfigurator
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.arguments.multiple
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.flag
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.types.file
+import com.squareup.kotlinpoet.FileSpec
+import java.io.File
+
+/** Generates plans from recipes. */
+class Recipe2Plan : CliktCommand(
+    help = """Generate plans from recipes.
+    
+    This script reads serialized manifests and generates Kotlin files with [Plan] classes.""",
+    printHelpOnEmptyArgs = true
+) {
+    val outdir by option(help = "output directory; defaults to '.'").file(fileOkay = false)
+    // TODO(161994250): Package should be derived from proto
```

Comment:
Nit: It's great there's a bug for the todo, but the typical pattern is to use `b/` prefix as well.

Reply:
Ah, thank you.